examples/requirements.txt

```diff
@@ -9,3 +9,4 @@
 Adafruit-Blinka
 adafruit-blinka-displayio
 blinka-displayio-pygamedisplay
+adafruit-fancyled
```

Comment:
This library failed to install when I tried. It should be referenced as `adafruit-circuitpython-fancyled`

Reply:
Thank you, I was doubtful. I check in my packages and your reference did appear available. Solved in https://github.com/adafruit/Adafruit_CircuitPython_ProgressBar/pull/29/commits/a6c291c666a0f8eb34f421a4372e35fa46d58a24

Reply:
Excellent! Great job :)